src/main/java/Task.java

```diff
@@ -0,0 +1,56 @@
+public class Task {
+
+    /**
+     * The description of the Task
+     */
+    protected String descr; //descr as an abbreviation for description
```

Comment:
I think it would be better if you just named it description, no need to abbreviate it and still comment on it